### PR TITLE
Fix build with qt6

### PR DESCRIPTION
**Pull request by jmlich**

### CODE REVIEW

File: src/alarmsbackendmodel_p.cpp [hunk 1]
```diff
@@ -170,7 +170,11 @@ void AlarmsBackendModelPriv::alarmUpdated(AlarmObject *alarm)
     if (currentRow >= 0)
         alarms.removeAt(currentRow);
 
+#if QT_VERSION >= QT_VERSION_CHECK(6, 0, 0)
+    QList<AlarmObject*>::iterator it = std::lower_bound(alarms.begin(), alarms.end(), alarm, alarmSort);
```
Comment (rainemak) [inline src/alarmsbackendmodel_p.cpp:173]: This would work for quite old Qt versions. I'd suggest just to replace qLowerBound with std::lower_bound.

**Comment by rainemak** [comment]
Minor nitpick to consider. Conceptually this looks good.

File: src/alarmsbackendmodel_p.cpp (hunk 1 above)
Comment (jmlich) [reply src/alarmsbackendmodel_p.cpp:173]: removed

**Review by rainemak — APPROVED** [review]
LGTM

**Pull request merged**